tensorflow_model_optimization/g3doc/guide/roadmap.md

```diff
@@ -20,7 +20,11 @@
     [Launched](https://blog.tensorflow.org/2019/05/tf-model-optimization-toolkit-pruning-API.html)
 *   Sparse model execution support in TensorFlow Lite --
     [WIP](https://github.com/tensorflow/model-optimization/issues/173)
-*   Weight clustering API
+
+## Weight clustering
+
+*   During-training weight clustering --
+    [Launched](https://blog.tensorflow.org/...)
```

Comment:
I guess this will be updated once the blog is published.

Reply:
Yes

Reply:
Will merge only after the publishing.